tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cuda102.yml

```diff
@@ -1,15 +1,14 @@
 trigger: none
 
 stages:
-- template: templates/py-packaging-stage.yml
+- template: templates/py-packaging-training-cuda-stage.yml
   parameters:
-    build_py_parameters: --enable_training --update --build
-    enable_linux_cpu: false
-    enable_linux_gpu: false
-    enable_linux_gpu_training_cu102: true
-    enable_linux_gpu_training_cu111: false
-    enable_linux_rocm_training: false
-    enable_windows_cpu: false
-    enable_windows_gpu: false
-    enable_mac_cpu: false
-    enable_linux_arm: false
+    build_py_parameters: --enable_training --update --build 
+    torch_version: '1.11.0'
+    opset_version: '14'
+    cuda_version: '10.2'
+    gcc_version: 8
+    cmake_cuda_architectures: 35;37;50;52;60;61;70
```

Comment:
"cmake_cuda_architectures", this is also something what you may want to discuss with the PTC team. Anything not in this list is not guaranteed to work(it might be an onnx runtime unique problem).

Reply:
+1 let's check the requirement with PTCA. If they don't have any specific requirements, I suggest removing 35 and adding 75 and 80 as these capabilities apply to Turing and Ampere architectures.